python-avd/pyavd/_eos_designs/structured_config/connected_endpoints/ethernet_interfaces.py

```diff
@@ -201,6 +199,8 @@ def _get_ethernet_interface_cfg(
             peer_type=connected_endpoint.type,
             port_profile=adapter.profile,
             peer_key=connected_endpoint._internal_data.context,
+            validate_state=None if (adapter.validate_state if adapter.validate_state is not None else True) else False,
+            validate_lldp=None if (adapter.validate_lldp if adapter.validate_lldp is not None else True) else False,
```

Comment:
let's simplify the quite tricky nested expression :) (the conclusion looing at this was that I need more ☕ )
```suggestion
            validate_state=False if adapter.validate_state is False else None,
            validate_lldp=False if adapter.validate_lldp is False else None,
```

Reply:
same for port-channel